dotnet/Workloads/SignList.targets

```diff
@@ -0,0 +1,53 @@
+<Project>
+  <!-- Extensions for signing nested .zip files for https://github.com/xamarin/yaml-templates/blob/e0f3cdce6210e05495188def8c695372b64ada1d/sign-artifacts/steps/v2-SignFiles.proj -->
+  <Target Name="_UnzipNestedZips"
+      BeforeTargets="_CalculateItemsToSign">
+    <PropertyGroup>
+      <_NestedZipExtractionDir>$(_WorkingDir)nested\</_NestedZipExtractionDir>
+    </PropertyGroup>
+
+    <ItemGroup>
+      <_NestedBrokerZip Include="$(_WorkingDir)**\Broker.zip" />
+      <_NestedBuildZip Include="$(_WorkingDir)**\Build.zip" />
+      <_NestediOSAppZip Include="$(_WorkingDir)**\Xamarin.PreBuilt.iOS.app.zip" />
+    </ItemGroup>
```

Comment:
What sets `$(_WorkingDir)`? Seems like this is blank and would just use the current directory?

Reply:
This file will be imported after the PropertyGroup that sets it in https://github.com/xamarin/yaml-templates/blob/e0f3cdce6210e05495188def8c695372b64ada1d/sign-artifacts/steps/v2-SignFiles.proj#L13